js/src/hud.js

```diff
@@ -12,6 +14,22 @@ function updateHud(ctx) {
   ctx.fillText(player_module.getHealth(), 68, 60);
 }
 
+function startScreen(ctx) {
+  ctx.font = 'bold 50px VT323';
+  ctx.fillText('Canvas Shooter', WIDTH / 2 - 150, HEIGHT / 2);
+  ctx.font = 'bold 20px VT323';
+  ctx.fillText('Hit SPACE to Play', WIDTH / 2 - 56, HEIGHT / 2 + 30);
+  ctx.fillText('Use arrow keys to move', WIDTH / 2 - 100, HEIGHT / 2 + 60);
+  ctx.fillText('Use the x key to shoot', WIDTH / 2 - 100, HEIGHT / 2 + 90);
+}
+
+function deathScreen(ctx) {
+  ctx.fillText('Game Over!', WIDTH/2 - 50, HEIGHT / 2);
+  ctx.fillText('Press SPACE to continue', 252, (HEIGHT / 2) + 35);
+}
+
 module.exports = {
-  updateHud : updateHud
+  updateHud : updateHud,
+  startScreen : startScreen,
+  deathScreen : deathScreen
 }
```

Comment:
If you're going with using semi-colons then note that your module.exports assignment statement 'needs' one too.  Although, of course JavaScript will insert one for you at compile time because its nice like that.

Reply:
Nice one. Good to know!